src/mono/wasm/Makefile

```diff
@@ -33,7 +33,7 @@ all: build-native icu-files source-files header-files
 #  If EMSDK_PATH is not set by the caller, download and setup a local emsdk install.
 #
 
-EMSCRIPTEN_VERSION=2.0.11
+EMSCRIPTEN_VERSION=2.0.12
```

Comment:
this will only update the version used by local developer builds, if you want to update the one used in CI you need to update platform-matrix.yml as described in https://github.com/dotnet/runtime/blob/master/docs/workflow/building/libraries/webassembly-instructions.md#updating-emscripten-version-in-docker-image

Reply:
It is already updated as per the comments.  This is to synchronize the versions

Reply:
https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/src/ubuntu/18.04/webassembly/Dockerfile#L19